geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/extract/TimeDimensionExtractor.java

```diff
@@ -0,0 +1,52 @@
+package mil.nga.giat.geowave.analytics.extract;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import mil.nga.giat.geowave.store.TimeUtils;
+
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.type.AttributeDescriptor;
+
+/**
+ * 
+ * A default implementation that averages all time attributes.
+ * 
+ */
+public class TimeDimensionExtractor extends
```

Comment:
my assumption is this is intended to be around for the eventuality of using time with analytics, but its not used yet...correct, or not?  Unlike my previous comment, I can see how this is worth having for the future, just not seeing it used now?